src/index.js

```diff
@@ -77,31 +79,41 @@ class SVGInline extends Component {
     const svgClasses = classes
       .split(" ")
       .join(classSuffix + " ") + classSuffix
-
+    let svgStr = SVGInline.cleanupSvg(svg, cleanup)
+      .replace(
+        /<svg/,
+        `<svg class="${ svgClasses }"` +
+        (
+          fill
+          ? ` fill="${ fill }"`
+          : ""
+        ) +
+        (
+          width || height
+          ? " style=\"" +
+              (width ? `width: ${width};` : "") +
+              (height ? `height: ${height};` : "") +
+            "\""
+          : ""
+        )
+      )
+    if(title && id) {
```

Comment:
Does not make sense to adjust output for each property individually?

Reply:
I could do it that way but 1) it wouldn't make much sense to have an id if the is no title provided and 2) it'd have to do yet another regex search to insert the id and aria-labelledby attrs (and would have to do basically the same check). The real a11y value is when both are present. 
Maybe it'd be better to rename the `id` prop to `titleId` to better clarify that this is used in the title element, not an id for the SVG itself.

Reply:
I just read about this via https://developer.paciellogroup.com/blog/2013/12/using-aria-enhance-svg-accessibility/

What about just having a single `accessibilityLabel` (which will be title text node) and generating an id directly? I don't see a real value of handling an idea by hand (as the only thing important is the value (title).
We could also imagine add `accessibilityRole` (and `accessibilityDesc`?)

My idea about this name is to match react-native(-web) style.
See https://facebook.github.io/react-native/docs/accessibility.html and https://github.com/necolas/react-native-web/blob/master/website/guides/accessibility.md

Reply:
`accessibilityLabel` is a much better name. It would make a lot of sense to add `role="img"` by default if an accessibilityLabel is provided. That would add the `accessibilityRole` functionality without exposing it as a prop; and I can't think of an instance where one might want an SVG to be identified as anything but an img. `accessibilityDesc` would be easy to add as well.

As for autogenerating ids for this... Would a simple id like `SVGInline-${i}-title` be acceptable, where `i` is substituted by a global count on the `SVGInline` object?

Reply:
Everything you said LGTM.